src/main/java/com/sapozhnykov/dao/order/impl/OrderDaoImpl.java

```diff
@@ -8,17 +8,43 @@
 import java.util.List;
 
 public class OrderDaoImpl implements OrderDao {
+    private static OrderDao orderDao = new OrderDaoImpl();
+
     private List<Order> orders = new ArrayList<>();
     private static long tempID = 1;
 
+    private OrderDaoImpl() { }
+
     @Override
     public boolean add(long clientId, ArrayList<Product> products) {
         Order order = new Order(tempID++, clientId, products);
         return orders.add(order);
     }
 
+    @Override
+    public boolean deleteById(long id) {
+        Order tempOrder;
+
+        tempOrder = getById(id);
```

Comment:
Please, merge declaration and initialization

Reply:
Fixed